src/components/shared/forms/styles/pill.scss

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+$pillHeight: 30px;
+$pillMinWidth: 127px;
+$pillIconSize: 14px;
+.pill {
+  display: flex;
+  flex-shrink: 0;
+  justify-content: space-between;
+  border: 0;
+  align-items: center;
+  text-align: left;
+  float: right;
+  white-space: nowrap;
+  margin: 7px 7px 0px 0px;
+  cursor: pointer;
+  font-family: inherit;
+  @include rem-fallback(padding, 0px, $pillHeight/4);
+  @include rem-fallback(min-width, $pillMinWidth);
+  p{
+    padding: 0px;
+    margin: 0px;
+  }
+  div{
+    float:right;
+  }
+
+  .pill-icon {
+    @include rem-fallback(height, $pillHeight);
```

Comment:
Generally - and this is a nit - I place includes after completely static definitions unless there is some explicit reason not to (like when the static declarations are needed to override something from the include).
```
.pill-icon {
  float: right;
  position: relative;
  @include rem-fallback(height, $pillHeight);
  @include rem-fallback(height, $pillHeight);
}
```

Reply:
Changed.